### DISCUSSION TITLE
Canceling copilot

### DISCUSSION THREAD

**Discussion by dfclark3** [community/community#146170] [Copilot Conversations]
### Select Topic Area

Question

### Body

I have not found how to cancel copilot. I searched and googled and hunted around the GitHub website. Can anyone point me to it? Thanks

**Comment by Akash1134** [comment]
Hi @dfclark3 ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by gtokman** [reply]
@Akash1134 Are you not able to cancel GH copilot from the GitHub site?

**Reply by pl-csci** [reply]
Why would you need to open a ticket to cancel?  Why isn't it obvious?
